product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/monitoring_using_pemx/index.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: "Monitoring using PEMx"
+---
+
+PEMx is an integrated monitoring and alerting solution within BigAnimal. It is specially designed to monitor the performance of the Postgres clusters. PEMx actively monitors various metrics of the Postgres clusters and triggers alert as the defined thresholds are exceeded. This solution smoothly integrates into the BigAnimal ecosystem, utilizing the vast amount of metric data that is gathered from Postgres clusters and their underlying infrastructure across all the regions. 
```

Comment:
We can consider PEMx a passive alert solution. We need to inspect the cluster to be aware of a metric over the threshold. So currently, we do not provide any active function that can notify the user that a cluster is unhealthy as said, we just highlight the charts if they are inspected by the users. I don't know if this is something that we need to explain to the users or keep it a bit general is fine. What do you think? @EnterpriseDB/pnxt

Reply:
Personally, I'm in favor with the current approach as the sections `Features for both types of charts` and `Charts error state` already explain how to detect cluster issues through the error icon in the chart. If notifications are not mentioned in the user document, I guess they will simply assume that are not available.

Reply:
Good I totally agree with you 😄

Reply:
No changes done as you are in favour of current approach